hudi-common/src/main/java/org/apache/hudi/common/config/HoodieReaderConfig.java

```diff
@@ -61,7 +61,7 @@ public class HoodieReaderConfig extends HoodieConfig {
 
   public static final ConfigProperty<Boolean> FILE_GROUP_READER_ENABLED = ConfigProperty
       .key("hoodie.file.group.reader.enabled")
-      .defaultValue(true)
+      .defaultValue(false)
```

Comment:
@nsivabalan How do I fetch the table version here in the infer function? I think it only allows hoodie config as parameter.

Reply:
Fixed the usage of these configs